.gitignore

```diff
@@ -2,3 +2,4 @@
 composer.phar
 composer.lock
 .DS_Store
+/.phpunit.result.cache
```

Comment:
Please add the `.idea` folder as an ignore for the project, doesn't make sense to commit these as we would via with a team perspective https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore